README.md

```diff
@@ -58,6 +58,8 @@ Object with initial settings:
 
 * **callback**:   *function* that call when countdown end, default **null**;
```

Comment:
As per your application, the time duration of a tick is completely configurable. If a user specifies a tick to be 1500ms, then a 3 step timer callback will execute every 4500ms.